server.js

```diff
@@ -5768,6 +5768,48 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Jetbrains Plugins repository integration
+camp.route(/^\/jetbrainsplugins\/([^\/]+)\/(d)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var pluginId = match[1];
+  var type = match[2];
+  var format = match[3];
+  var badgeData = getBadgeData('jetbrains plugin', data);
+  var url = 'https://plugins.jetbrains.com/plugins/list?pluginId=' + pluginId;
+
+  request(url, function(err, res, buffer) {
+    if (err) {
+      badgeData.text[1] = 'inaccessible';
```

Comment:
Unfortunately, XML API does not return 404 when plugin is not found. Sorry, I didn't check this before. 

    > curl "https://plugins.jetbrains.com/plugins/list?pluginId=PLUGIN_NOT_FOUND"      
    <?xml version="1.0" encoding="UTF-8"?><plugin-repository></plugin-repository>